dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -264,6 +264,8 @@
 			</_CustomLinkerOptions>
 			<_ExtraTrimmerArgs>$(_ExtraTrimmerArgs) --custom-data "LinkerOptionsFile=$(_CustomLinkerOptionsFile)"</_ExtraTrimmerArgs>
 
+			<_ExtraTrimmerArgs Condition="'$(_SdkIsSimulator)' != 'true' And '$(_PlatformName)' != 'macOS'">$(_ExtraTrimmerArgs) --feature ObjCRuntime.Runtime.DeviceSupport true</_ExtraTrimmerArgs>
```

Comment:
What about Mac Catalyst?

Reply:
ah, missing projects for catalyst [dotnet] :(